python/Ganga/testlib/GangaUnitTest.py

```diff
@@ -186,9 +214,7 @@ def stop_ganga():
     ShutdownManager._ganga_run_exitfuncs()
 
     # Undo any manual editing of the config and revert to defaults
-    from Ganga.Utility.Config import allConfigs
-    for package in allConfigs.values():
-        package.revertToDefaultOptions()
+    clear_config()
```

Comment:
Again, it would be nice to see this included somwhere in the normal Ganga shutdown rather than here, I would prefer to see this as minimal as possible. I can't think why this is a special case